### ISSUE TITLE
Release a new version of XRouting

### ISSUE THREAD

**Issue by Jako** [SEDAdigital/XRouting#36]
... in MODX extras. There were some patches after 1.4.1.

**Comment by lottaar** [comment]
I'm having trouble getting XRouting 1.4.1 to work with Babel 3.0.0. When I install XRouting the Babel links stop working. Is there a newer version of XRouting that might fix this? Where can I find it?

**Comment by lottaar** [comment]
Right, I found version 1.5.1 so I pasted that into the plugin in MODX. It fixed the issue I had with the Babel links not working (/en, /fr etc), but instead the context which uses another domain isn't working, it just goes to the web context. With 1.4.1 it was the other way around...

Is there an even later version than 1.5.1 somewhere I could try? Or perhaps something is wrong in my .htaccess...

**Comment by christianseel** [comment]
There is no newer version than that. Did you had luck fixing your issues? Otherwise could you provide a full description of your issues, maybe better in a new dedicated issue?

**Comment by lottaar** [comment]
I managed to solve it with the help of the awesome MODX community, so it's working now. The root of the problem was a conflict with another gateway snippet plus Chrome caching the .htaccess file.

**Comment by travisbotello** [comment]
Bumping this...it s still installing the old version without new features/fixes. Please update!!

**Comment by Jako** [comment]
@christianseel There are [some patches](https://github.com/SEDAdigital/XRouting/commits/master) in this repository after version 1.4.1 released in May 20, 2015 which were never released or packaged. Please release them in a new version on MODX Extras.
